tinymce/views.py

```diff
@@ -4,6 +4,11 @@
 import json
 import logging
 
+import django
+if django.VERSION < (2,):
+    import warnings
+    warnings.warn("Update to Django 2.0", DeprecationWarning)
```

Comment:
We should change the message to something like: `Support for Django < 2.0 will be removed soon, please upgrade your project to use Django 2.0`